src/serac/physics/tests/solid_multi_material.cpp

```diff
@@ -198,7 +198,7 @@ TEST(Solid, MultiMaterialWithState)
   constexpr double Hi = E_right / 3.6;
   constexpr double sigma_y = 0.75 * applied_stress;
 
-  Hardening hardening{.sigma_y = sigma_y, .Hi = Hi};
+  Hardening hardening{.sigma_y = sigma_y, .Hi = Hi, .eta = 0};
```

Comment:
```suggestion
  Hardening hardening{.sigma_y = sigma_y, .Hi = Hi, .eta = 0.0};
```

Just so it's clear it's a double